addons/@mlabs-haskell/gd-cardano/bin/gdexample.gdextension

```diff
@@ -0,0 +1,23 @@
+[configuration]
+
+entry_symbol = "gdext_rust_init"
+compatibility_minimum = "4.1"
+
+[libraries]
+
+macos.debug = "res://addons/@mlabs-haskell/gd-cardano/bin/libcsl_godot.macos.template_debug.framework"
```

Comment:
- this file should be named something like libcsl_godot.gdextension or godot-cardano.gdextension
- the path should probably be `addons/@mlabs-haskell/godot-cardano` or is `gd-` a convention? we can also rename the repo

Reply:
I agree about renaming the file. I think `libcsl_godot.gdextension` makes sense.

> is `gd-` a convention?

I'm not sure about that, I can rename it to `godot-cardano` for now.